src/test/endorsement-chain.spec.js

```diff
@@ -5,7 +5,12 @@ fixture("Endorsement Chain Rendering").page`http://localhost:3000`;
 
 const ViewEndorsementChainButton = Selector("#endorsement-chain-button");
 const EndorsementChainTitle = Selector(".endorsement-chain-title");
-const EndorsementChainAddress = Selector("div.address").withText("0x8e87c7cEc2D4464119C937bfef3398ebb1d9452e");
+const EndorsementChainAddress1 = Selector("div.address").withText("0xFC6e365B926166d0D69bF336d03164FB301D6C41");
+const EndorsementChainAddress2 = Selector("div.address").withText("0x1Dc271EaE22a83c9670571d1a206043E8a420fdE");
+const DocumentIssuedAction = Selector("div.name").withText("Document has been issued");
```

Comment:
i think can be looser, omit the div selector

Reply:
ok